UnicornRender/video/include/unicorn/video/Texture.hpp

```diff
@@ -44,7 +46,7 @@ class Texture
     /**
      * @brief Remove texture data from memory
      */
-    UNICORN_EXPORT void Delete();
+    UNICORN_EXPORT void FreeData();
```

Comment:
Why is this method public? When should it be called by the user?

Reply:
This methods removes bytes of texture from RAM, not texture as C++ object as for desctructor.
So user can create texture
LoadData
FreeData
LoadData
destruct texture
Thats idea

Reply:
Can `LoadData()` call this method? why it should be called by the user?